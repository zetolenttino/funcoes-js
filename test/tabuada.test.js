const tabuada = require('../src/tabuada')

describe('tabuada', ()=>{
    it('tabuada de soma do numero 5', ()=>{
        expect(tabuada(5, function (a,b) {
            return a+b
        } )).toMatchSnapshot()
    })
    it('tabuada de multiplicação do numero 5', ()=>{
        expect(tabuada(5, function (a,b) {
            return a*b
        })).toMatchSnapshot()
    })
})

/*

arraw function - flecha - lambda
    (a,b) => a+b

    anonymous function - função sem nome

function (a,b) {
    return a+b
}

*/