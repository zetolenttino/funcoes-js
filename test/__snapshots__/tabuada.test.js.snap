// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tabuada tabuada de multiplicação do numero 5 1`] = `
"
5
10
15
20
25
30
35
40
45
50"
`;

exports[`tabuada tabuada de soma do numero 5 1`] = `
"
6
7
8
9
10
11
12
13
14
15"
`;
